app/services/mapquest_service.rb

```diff
@@ -8,6 +8,16 @@ def get_lat_long(location)
     json[:results][0][:locations][0][:latLng]
   end
 
+  def road_trip(origin, destination)
```

Comment:
cleaner endpoint. For some reason had the origin and destination in the endpoint which makes no sense. Rookie mistake